_config.yml

```diff
@@ -28,6 +28,10 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://dpc.cms.gov" # the base hostname & protocol for your site, e.g. http://example.com
 cssurl: /assets/css/static-main.css
 logourl: /assets/images/top-nav-heart.svg
+prod_register: "https://prod.dpc.cms.gov/users/sign_up"
```

Comment:
It's generally an anti-pattern to define variables (or in this case config params) we're not using.  In particular, I'm not sure yet whether we'll want a signup link in production, so I question including this one.

Reply:
It's easy enough to remove and then add again. You'll have to approve the PR again.

Reply:
@dhgreene Please approve again! 🙏